carbon-mb/components/transports/mqtt/src/main/java/org/wso2/carbon/andes/transports/mqtt/broker/BrokerVersion.java

```diff
@@ -43,7 +43,7 @@
     /**
      * We maintain the instance of the broker which corresponds to its version
      */
-    private Broker broker;
+    private transient Broker broker;
```

Comment:
BrokerVersion is an 'enum' which is inherently serializable [1]. Broker interface is a field defined in the enum which holds objects which do not require serialization.

However, findbugs identify this and pops out a warning, a similar discussion on this could be found in [2] on java Set<?> interface in its serialized context. In this case since we do not serialize the concrete implementation of the broker we explicitly specify that the implementation of the broker would be transient. 

[1] http://docs.oracle.com/javase/6/docs/api/java/lang/Enum.html 
[2] http://stackoverflow.com/questions/4861228/how-to-handle-a-findbugs-non-transient-non-serializable-instance-field-in-seria